api_server/swagger/firecracker.yaml

```diff
@@ -501,6 +528,21 @@ definitions:
         type: string
         description: Host level path for the guest drive
 
+  PartialNetworkInterface:
```

Comment:
:( this is sad. Maybe create an issue with the "Technical Debt" label so we can re-design this in the future.